### PR TITLE
depends, docker: fix bintray links

### PR DESCRIPTION
**Pull request by selsta**
Bintray is shutting down.

### CODE REVIEW

**Comment by hyc** [comment]
I see boost versions 1.64, 1.68, and 1.70 in these files. Shouldn't we consolidate on a single version?

**Comment by selsta** [comment]
Should this be done in a separate PR? The goal of this one is just to fix offline links.

File: Dockerfile [hunk 1]
```diff
@@ -46,7 +46,7 @@ ARG BOOST_VERSION=1_70_0
 ARG BOOST_VERSION_DOT=1.70.0
 ARG BOOST_HASH=430ae8354789de4fd19ee52f3b1f739e1fba576f0aded0897c3c2bc00fb38778
 RUN set -ex \
-    && curl -s -L -o  boost_${BOOST_VERSION}.tar.bz2 https://dl.bintray.com/boostorg/release/${BOOST_VERSION_DOT}/source/boost_${BOOST_VERSION}.tar.bz2 \
+    && curl -s -L -o  boost_${BOOST_VERSION}.tar.bz2 https://downloads.sourceforge.net/project/boost/boost/${BOOST_VERSION_DOT}/boost_${BOOST_VERSION_DOT}.tar.bz2 \
```
Comment (mj-xmr) [inline Dockerfile:49]: Should the second instance also be `...DOT` ?

Comment (selsta) [reply Dockerfile:49]: Thanks.

**Review by mj-xmr — APPROVED** [review]

**Review by xiphon — APPROVED** [review]

**Pull request merged**